pol-core/pol/module/uomod2.cpp

```diff
@@ -1589,6 +1589,9 @@ BObjectImp* UOExecutorModule::mf_SendTextEntryGump()
     return new BError( "No client attached" );
   }
 
+  if ( chr->client->gd->textentry_uoemod != nullptr )
```

Comment:
There are a few methods in Character:

has_active_prompt()
has_active_gump()
etc...

Maybe this would fit better as a similar method?